.ci/pytorch/macos-test.sh

```diff
@@ -244,13 +244,13 @@ test_torchbench_smoketest() {
       PYTHONPATH="$(pwd)"/torchbench python benchmarks/dynamo/torchbench.py \
         --performance --only "$model" --backend "$backend" --inference --devices "$device" "$dtype_arg" \
         --output "$TEST_REPORTS_DIR/inductor_${backend}_torchbench_${dtype}_inference_${device}_performance.csv" || true
-      if [ "$backend" == "inductor" ]; then
+      if [ "$backend" == "inductor" ] || [ "$backend" == "export-aot-inductor" ]; then
```

Comment:
In the current state, the benchmark hasn't been run yet https://github.com/pytorch/pytorch/actions/runs/16991892110/job/48174762971#step:16:6785.  Passing e`xport-aot-inductor` as a backend turns out to be the wrong way.  From what I see, it's a flag `--export-aot-inductor`, and isn't passed as a compiler backend parameter.  The existing inductor benchmark refers to it as `aot_inductor`, but just on the output filename https://github.com/pytorch/pytorch/blob/0d28d12b11ddc59b5677775e03ff2f26b359f861/.ci/pytorch/test.sh#L698C52-L698C64.  I think we need to do the same thing here, passing `--export-aot-inductor` and using `aot_inductor` in the filename (for consistency)